.github/workflows/ci.yml

```diff
@@ -122,7 +122,20 @@ jobs:
         if: ${{ !matrix.n3ds }}
         run: |
           cd build
-          cpack .
+          success=0
+          max_tries=10
+          for i in $(seq $max_tries); do
+            cpack . && success=1
```

Comment:
[What's the reason behind all these warnings on ci?](https://github.com/isledecomp/isle-portable/actions/runs/16146192831/job/45565554979?pr=555#step:13:25)